pyqt-apps/siriushla/as_ps_control/detail_widget/PSDetailWidget.py

```diff
@@ -758,20 +758,30 @@ def _siggenLayout(self):
         layout.addWidget(self.tab_siggen, 0, 0)
         return layout
 
-    def _sofbModeLayout(self):
+    def _iocModeLayout(self):
         sofb_mode_sel = self._prefixed_psname + ':SOFBMode-Sel'
         sofb_mode_sts = self._prefixed_psname + ':SOFBMode-Sts'
 
-        sofb_mode_label = QLabel('Enable', self)
+        sofb_mode_label = QLabel('SOFB', self)
         sofb_mode_btn = PyDMStateButton(self, sofb_mode_sel)
         sofb_mode_led = SiriusLedState(self, sofb_mode_sts)
 
+        idff_mode_sel = self._prefixed_psname + ':IDFFMode-Sel'
+        idff_mode_sts = self._prefixed_psname + ':IDFFMode-Sts'
+
+        idff_mode_label = QLabel('IDFF', self)
+        idff_mode_btn = PyDMStateButton(self, sofb_mode_sel)
+        idff_mode_led = SiriusLedState(self, sofb_mode_sts)
```

Comment:
wrong channel names: sofb_mode_sel and sofb_mode_sts

Reply:
thanks, @anacso17 . fixed!